geostore/populate_catalog/task.py

```diff
@@ -95,8 +95,4 @@ def handle_message(metadata_key: str) -> None:
     if root_catalog.get_child(dataset_metadata.id) is None:
         root_catalog.add_child(child=dataset_metadata, strategy=GeostoreSTACLayoutStrategy())
 
-    root_catalog.normalize_hrefs(
```

Comment:
We do want to normalize hrefs across the Geostore (hopefully we have a test to enforce that). Have you checked that we have code and/or a test for that?

Reply:
We definitely don't need to normalize every href every time we add a dataset though, just need to do the catalog.json and all the STAC file in the child that is being added. Can ignore all the siblings.

Reply:
This was addressed in https://github.com/stac-utils/pystac/pull/294 (if I read the PR correctly). Nevertheless, tests are failing which means more work is needed to figure out what is needed to fix this issue.